src/bun.js/node/node_util_binding.zig

```diff
@@ -105,3 +105,70 @@ pub fn internalErrorName(globalThis: *JSC.JSGlobalObject, callframe: *JSC.CallFr
     var fmtstring = bun.String.createFormat("Unknown system error {d}", .{err_int}) catch bun.outOfMemory();
     return fmtstring.transferToJS(globalThis);
 }
+
+/// `extractedSplitNewLines` for ASCII/Latin1 strings. Panics if passed a non-string.
+//Returns `undefined` if param is utf8 or utf16 and not fully ascii.
+///
+/// ```js
+/// // util.js
+/// const extractedNewLineRe = new RegExp("(?<=\\n)");
+/// extractedSplitNewLines = value => RegExpPrototypeSymbolSplit(extractedNewLineRe, value);
+/// ```
+pub fn extractedSplitNewLinesFastPathStringsOnly(globalThis: *JSC.JSGlobalObject, callframe: *JSC.CallFrame) bun.JSError!JSC.JSValue {
```

Comment:
this might be a good place to use bindgen, though something we should add to that is a way to turn all validations into assertions (internal APIs like this)

```zig
pub fn extractedSplitNewLinesFastPathStringsOnly(global: *JSC.JSGlobalObject, str: bun.String) !JSC.JSValue {
    // ...
}
```
Create `node_util_binding.bind.ts`
```
import { fn, t } from 'bindgen';
export const extractedSplitNewLinesFastPathStringsOnly = fn({
  args: {
    global: t.globalObject,
    str: t.DOMString,
  },
  ret: t.any,
})
```
in JavaScript builtins
```diff
-$newZigFunction("node_util_binding.zig", "extractedSplitNewLinesFastPathStringsOnly", 1);
+$bindgenFn("node_util_binding.bind.ts", "extractedSplitNewLinesFastPathStringsOnly");
```
I wont block the pr on this point, but i think your interest in bindgen would want to use it here.